features/cli.feature

```diff
@@ -3,6 +3,21 @@ Feature: Command line interface
   As a person who wants to run features
   I want to run Cucumber on the command line
 
+  Scenario: run with no features directory
+    When I run cucumber.js with `-f summary`
+    Then the error output contains the text:
+    """
+    No such file or directory - features. You can use `cucumber --init` to get started.
+    """
+
+  Scenario: run a nonexistant feature file
+    Given a directory named "features" 
+    When I run cucumber.js with `-f summary features/a.feature`
+    Then the error output contains the text:
+    """
+    No such file or directory - features/a.feature. You can use `cucumber --init` to get started.
+    """
```

Comment:
I'm not certain about this, as this happens to me when I rename a feature file. Do we really need to suggest running `--init`? We could just add some documentation about it in the README